### PR TITLE
Bitnami.XAMPP.7.4 version 7.4.29

### PR DESCRIPTION
**Pull request by EchoEllet**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62812)

### CODE REVIEW

File: manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Bitnami.XAMPP.7.4
```
Comment (Trenly) [inline manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.installer.yaml:4]:
```suggestion
PackageIdentifier: ApacheFriends.XAMPP
```

File: manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Bitnami.XAMPP.7.4
```
Comment (Trenly) [inline manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: ApacheFriends.XAMPP
```

File: manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Bitnami.XAMPP.7.4
```
Comment (Trenly) [inline manifests/b/Bitnami/XAMPP/7/4/7.4.29/Bitnami.XAMPP.7.4.yaml:4]:
```suggestion
PackageIdentifier: ApacheFriends.XAMPP
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Package should use already existing identifier. Metadata should be taken from the already existing identifier

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**